functions/src/definitions/eventHandlers/userHandlers.ts

```diff
@@ -851,6 +852,16 @@ async function addInstanceToDb(
   instanceRef: FirebaseFirestore.DocumentReference,
   instanceUpdateObj: Object
 ) {
+  // Repost instance to admin channel
+  // @ts-ignore
+  if (instanceUpdateObj.text) {
+    await repostText(id, instanceUpdateObj)
```

Comment:
I think best to avoid repostText/repostImage in userHandler, because userHandler will eventually be refactored to take messages from multiple sources, not just whatsapp.

Best to place the repost logic into the existing eventHandlers/onMessageUpdate and create a new eventHandlers/onMessageCreate to listen to DB changes. This will be the best place to put the repost cos it'll be agnostic to wherever the messages come from in the future. Basically we want to place the repost logic with the most fundamental construct, database object creation, rather than at the handling logic which is one step removed and might often change.